src/app/pages/admin-page/admin-page.component.html

```diff
@@ -0,0 +1,3 @@
+<p>Admin page works</p>
+
+<button (click)="goBack()">Back</button>
```

Comment:
use location.back(), there are no need in additional class method if there are no additional actions/calculations